testsrc/org/mozilla/javascript/tests/Test262SuiteTest.java

```diff
@@ -51,7 +51,7 @@
 
     private static final File testDir = new File("test262/test");
     private static final String testHarnessDir = "test262/harness/";
-    private static final String testProperties = "testsrc/test262.properties";
+    private static String testProperties = "testsrc/test262.properties";
```

Comment:
Is this to remain `static`? I'm not familiar with the behavior of JUnit, but if the variable changes with each execution, I'm worried that `static` will be a problem.

Reply:
Sorry. It needs to be `static`.
https://github.com/mozilla/rhino/blob/a84308f16abd10276a42ce1b7b3211cc0c674a48/testsrc/org/mozilla/javascript/tests/Test262SuiteTest.java#L96

Reply:
You can make this work if you make testProperties final again, but don't assign a value in the initializer -- instead add an "else" down in your static block to assign the default value if the system property isn't there. This is exactly the same thing that we do in that same static block for OPT_LEVELS.

This minor but we're trying to be more careful and consistent with the little things across the codebase. Thanks!

Reply:
Changed